www/templates/html/MediaPlayer.tpl.php

```diff
@@ -6,13 +6,184 @@
 }
 ?>
 
+<?php if($context->media->getTextTracks()): ?>
+<script type="html/template" class="mh_transcript_template">
+
+        <div class="mh-caption-search">
+            <h6 class="wdn-sans-serif wdn-icon-search">
```

Comment:
Not sure why I didn't notice this before, but I think the `h6` element is not a good choice. Without knowledge of the document outline, trying to guess which heading number to use is bad. Let's just use some generic tags for now.